src/plugins/conceptPlugin.js

```diff
@@ -23,6 +23,22 @@ export default function createConceptPlugin(options = {}) {
   const fetchResource = (embed, accessToken, language) =>
     fetchConcept(embed, accessToken, language, options);
 
+  const getIframeProps = concept => {
+    return {
+      src: `https://liste.ndla.no/concepts/${concept.id}`,
```

Comment:
Usikker på denne, det var noe snakk om at lenken skulle være avhengig av miljø, men jeg vet ikke åssen jeg skulle fått miljøinformasjonen i article-converter

Reply:
Den må fort være avhengig av miljø ja.
Hvis du ser i config.js fila så ser du hvordan vi bygger opp domenet for api-url'en.
Kan gjøres på samme måte her :smile: